packages/next/src/build/babel/loader/get-config.ts

```diff
@@ -470,9 +486,9 @@ export default async function getConfig(
     loaderOptions: NextBabelLoaderOptions
     target: string
     filename: string
-    inputSourceMap?: object | null
```

Comment:
While babel's type annotations indicate that `null` is okay here, they have a runtime validator that checks that allows `undefined`, but not `null` 🙃